REUSE

SebaUbuntu opened the pull request.

SebaUbuntu committed:
Introduce logger utils

Change-Id: I2efdccd24f624f7c242669b89c8f53863601557b

SebaUbuntu committed:
Introduce license parsing

Change-Id: Ibeb12b9c92ca5d91c42ac159d65d3adc1d698246

SebaUbuntu committed:
Write SPDX header on libs/Android.bp

Change-Id: I8e35892f3ced15b788b7a68358c82517ff73ab5c

luk1337 commented on src/main/java/org/lineageos/generatebp/models/Artifact.kt:
```diff
@@ -61,40 +76,49 @@ data class Artifact(
     )
 
+    val reuseCopyrightFileContent by lazy {
+        assert(licenses.isNotEmpty()) {
```
assertions seem to not work here, maybe replace them with something else.

luk1337 replied:
require() seems to work.

luk1337 replied:
Pushed change that switches to it.

luk1337 committed:
Switch from assert() to require()

assert() requires that JVM has assertions enabled, which doesn't seem to
be a case when I run `./gradlew app:generateBp`.
